src/dev/code_coverage/shell_scripts/merge_jest_and_functional.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+EXTRACT_START_DIR=tmp/extracted_coverage
+EXTRACT_END_DIR=target/kibana-coverage
+COMBINED_EXRACT_DIR=/${EXTRACT_START_DIR}/${EXTRACT_END_DIR}
```

Comment:
s/EXRACT/EXTRACT :)

Reply:
Sure

Reply:
<img width="1138" alt="Screen Shot 2020-05-06 at 11 36 30 AM" src="https://user-images.githubusercontent.com/4858051/81209505-ded6c800-8f8d-11ea-80cf-969bd817b55a.png">
 
I do love using regex in my ide :)